oap-server/pom.xml

```diff
@@ -64,7 +64,7 @@
         <h2.version>1.4.196</h2.version>
         <commons-dbcp.version>1.4</commons-dbcp.version>
         <commons-io.version>2.6</commons-io.version>
-        <elasticsearch.version>6.3.2</elasticsearch.version>
+        <elasticsearch.version>6.8.10</elasticsearch.version>
```

Comment:
Could we don't do this? Once you did, I remember, you can't access 6.3-6.7 server any more.

Reply:
I notice the same thing, so just revert that commit.